packages/database/scripts/migrate.ts

```diff
@@ -0,0 +1,17 @@
+import {migrate} from 'drizzle-orm/mysql2/migrator';
+import db from '../src/db/connection';
+import {exit} from 'node:process';
+
+async function runMigrations() {
+  try {
+    await migrate(db, {
+      migrationsFolder: './../database/migrations',
+    });
+    exit();
```

Comment:
Is the exit required on a successful run?

Reply:
Yes, without it, the script will never stop. I tried it with `return`, but this did nothing. Only the `exit();` will finish the script when running `npm run db:migrate`.

Reply:
That's strange :) Do you know why the script never stops? After `migrate` there are no more tasks to complete, so the Node process should exit automatically. Is there another process running that keeps it from exiting? For example: is `ts-node` in the way?

Reply:
This is a bug in Drizzle.

See:

https://github.com/drizzle-team/drizzle-orm/issues/1222
https://www.reddit.com/r/node/comments/14zggaj/tsnode_does_not_exit/

`exit()` is a workaround.

Reply:
Interesting! Add a comment to the code, referring to the bug? Then we know where it comes from.